benchmarks/src/main/java/org/elasticsearch/benchmark/compute/operator/ValuesSourceReaderBenchmark.java

```diff
@@ -189,6 +189,11 @@ public String indexName() {
                     return "benchmark";
                 }
 
+                @Override
+                public boolean forStats() {
```

Comment:
We've considered alternative names, like `forDisplay` (opposite of for stats), but every choice has pros and cons, so I've left it like this for now, since this most closely matches current needs.

If we start using this for non-stats use cases, like spatial predicates that benefit from doc-values, we can change the name of the method at that point.

Reply:
Predicate and top-n run on the results of functions consuming the points will want it.